Xamarin.RevenueCat.iOS/Xamarin.RevenueCat.iOS.csproj

```diff
@@ -68,7 +68,28 @@
             <Kind>Framework</Kind>
             <SmartLink>False</SmartLink>
             <ForceLoad>True</ForceLoad>
+            <NeedsGccExceptionHandling>False</NeedsGccExceptionHandling>
```

Comment:
sure about that?

Reply:
yeah we don't need to link with gcc https://learn.microsoft.com/en-us/dotnet/api/objcruntime.linkwithattribute.needsgccexceptionhandling?view=xamarin-ios-sdk-12 and from here and from example I don't see `gcc_eh` is explicitly needed https://github.com/RevenueCat/purchases-ios